pos_default_quantity/static/src/js/pos.js

```diff
@@ -0,0 +1,49 @@
+/*
+  Copyright 2019 Coop IT Easy SCRLfs
+    Robin Keunen <robin@coopiteasy.be>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+odoo.define(
+    'pos_default_quantity.pos_default_quantity',
```

Comment:
Please name the `.js` file same as your `define`.

ie:
both `pos.js` and `odoo.define("pos_default_quantity.pos")`, for example